incubator/kafka/values.yaml

```diff
@@ -143,52 +143,100 @@ persistence:
   ##
   # storageClass:
 
-## Metrics
+jmx:
+
+  ## Rules to apply to the Kafka JMX Exporter
+  configMap:
+
+    ## Allows disabling the default configmap, note a configMap is needed
+    enabled: true
+
+    ## Allows setting values to generate confimap
+    overrideConfig: {}
+      # jmxUrl: service:jmx:rmi:///jndi/rmi://127.0.0.1:5555/jmxrmi
+      # lowercaseOutputName: true
+      # lowercaseOutputLabelNames: true
+      # ssl: false
+
+    ## Allows setting the name of the ConfigMap to use, note a configMap
```

Comment:
Can we change the wording here a bit -- it took me a read or two to parse what this meant. I think something along the lines of:

```
## If you would like to supply your own ConfigMap for JMX metrics, supply the name of that 
## ConfigMap as an `overrideName` here.
```